### PR TITLE
add test for TFHub export

### PR DESCRIPTION
**Pull request by vasudevgupta7**
@sayakpaul, I am exporting converted fine-tuned checkpoint for now as we less time left until the deadline (as discussed earlier). Added few tests in this PR to ensure I am exporting correct model to TFHub.

Update regarding training:
Training experiments are in progress and I have fixed some other bugs in `training-v3` branch and seeing lot of improvement in loss function. I will keep you updated as the results improve. You can track experiments on wandb ([here](https://wandb.ai/7vasudevgupta/gsoc-wav2vec2/runs/3d1jaati?workspace=user-7vasudevgupta)).

### CODE REVIEW

**Comment by sayakpaul** [comment]
Fair enough. Thanks for the update. Keep us posted.

@MorganR

**Comment by vasudevgupta7** [comment]
TFHub tests are in this PR only😅. Are you talking about something else??

**Review by sayakpaul — APPROVED** [review]
